docs/tutorials/uml/ComponentDiagram.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+!include <tupadr3/common>
+!include <office/Concepts/globe_internet>
+!include <office/Concepts/documents>
+!include <office/Users/user>
+!include style.puml
+
+Package " "<<Rectangle>>{
+    Class UI UI_COLOR
+    Class Logic LOGIC_COLOR
+    Class Storage STORAGE_COLOR
+    Class Model MODEL_COLOR
+    Class Main MODEL_COLOR_T1
+    Class Commons LOGIC_COLOR_T2
+    Class "Log Center" as Logs UI_COLOR_T2
+    Class Hidden #FFFFFF
+}
+Class "<$user>" as User MODEL_COLOR_T2
+Class "<$documents>" as File UI_COLOR_T1
+
+User -right-> UI
+UI -> Logic
+UI -right-> Model
+Logic -> Storage
+Logic -down-> Model
+
+
+Main -up-> Model
+Main -> Commons
+Main -> UI
+
+Logs -up- Commons
+Hidden .down.> Commons
+Hidden .down.> Commons
+Hidden .down.> Commons
+
+Storage .right.>File
+
+hide circle
```

Comment:
This would not be here if we applied DRY and used the ArchitectureDiagram.puml that's in master ;-)

Anyway I think this file is out of date.